### PR TITLE
Session cards GitHub fixes

### PR DESCRIPTION
**Pull request by EricMason24**
Converts the session overview style into cards and hopefully fixes github issues where a pull request didn't occur

### CODE REVIEW

File: src/components/Breakout.vue [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<template>
```
Comment (dborstelmann) [inline src/components/Breakout.vue]: No spaces in attributes

File: src/components/Breakout.vue [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  <el-card class = "Breakout" shadow="hover">
```
Comment (dborstelmann) [inline src/components/Breakout.vue]: Also don’t capitalize classes

File: src/components/Breakout.vue [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+export default {
+  name: "Breakout",
+  props: {
+    title:String,
+    time:String,
+    roomNumber:Number,
+    moreInfo:String
+  },
+  computed:{
+    computedTime: function(){
```
Comment (dborstelmann) [inline src/components/Breakout.vue]: Absolutely perfect.

File: src/components/Breakouts.vue [hunk 4]
```diff
@@ -1,54 +1,38 @@
...
+    <Breakout :title="breakout.title" :time="breakout.time" :roomNumber="breakout.room_num" moreInfo="www.google.com"  class = "Breakout"/>
+  </li>
+<!-- </el-row> -->
+</el-row>
 </template>
 
 <script>
 import axios from 'axios'
-import moment from 'moment'
+import Breakout from "@/components/Breakout.vue";
```
Comment (dborstelmann) [inline src/components/Breakouts.vue]: You should be able to just `import Breakout from “Breakout.vue”`

File: src/components/Breakouts.vue [hunk 5]
```diff
@@ -1,54 +1,38 @@
...
 export default {
   name: "Breakouts",
+  components: {
+    'Breakout':Breakout
```
Comment (dborstelmann) [inline src/components/Breakouts.vue]: Please use a space after your colons

File: src/components/Panels.vue [hunk 6]
```diff
@@ -1,54 +1,38 @@
...
+    <Panel :title="panel.title" :time="panel.time" :roomNumber="panel.room_num" moreInfo="www.google.com"  class = "Panel"/>
+  </li>
+<!-- </el-row> -->
+</el-row>
 </template>
 
 <script>
 import axios from 'axios'
-import moment from 'moment'
+import Panel from "@/components/Panel.vue";
```
Comment (dborstelmann) [inline src/components/Panels.vue]: Same as above

File: src/components/Person.vue [hunk 7]
```diff
@@ -1,13 +1,17 @@
...
-      <p>{{ company }}</p>
-      <p>{{ bio }}</p>
-    </div>
+        <el-card shadow="hover" :body-style="{ padding: '0px' }">
+          <div class="image-wrapper">
+            <img :src="imgSource" class="image">
+          </div>
+          <div style="padding: 10px;">
+            <span class="name">{{ name }}</span>
+            <div class="bottom clearfix">
```
Comment (dborstelmann) [inline src/components/Person.vue]: I made comments about this in the other pull request, is that not getting merged before this? I would still like those comments answered

Comment (EricMason24) [reply src/components/Person.vue]: This was in Soraya's request, which has not merged into staging yet -- this is the most recently updated version from her end

Comment (dborstelmann) [reply src/components/Person.vue]: Thanks.  I responded in #9

**Pull request merged**